packages/cloud-cognitive/src/components/Datagrid/styles/addons/_CustomizeColumnsModal.scss

```diff
@@ -46,3 +61,22 @@
   position: relative; // needed for the react-dnd, otherwise the drag preview will not work correctly
   overflow: auto;
 }
+
+#customize-columns-select-all {
+  padding-left: $spacing-08;
+}
+
+.bx--checkbox-label {
+  &.#{$block-class}__customize-columns-checkbox {
+    padding-left: $spacing-07;
+  }
+
+  &.#{$block-class}__customize-columns-select-all {
+    padding-left: $spacing-07;
+    @include carbon--font-weight('semibold');
+  }
+}
+
+#bx--modal-body--modal-2 {
```

Comment:
This id changes each time the modal is rendered so it's not a reliable selector here. You could potentially use something like:
```scss
.#{$block-class} .#{$block-class}__customize-columns-modal .#{$carbon-prefix}--modal-content {
...
}
```

Reply:
replaced `#bx--modal-body--modal-2` with `.#{$block-class} .#{$block-class}__customize-columns-modal .#{$carbon-prefix}--modal-content`